catalog/data_table.go

```diff
@@ -0,0 +1,24 @@
+package catalog
+
+import (
+	"context"
+
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/service/catalog"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type tableParams struct {
+	FullName string `json:"full_name" tf:"computed,optional`
```

Comment:
Maybe it's better to have two params: catalog & schema?
We use this approach in other data sources: https://github.com/databricks/terraform-provider-databricks/blob/main/catalog/data_views.go#L14

Reply:
The Databricks API takes a [single parameter for the fully qualified table name](https://docs.databricks.com/api/workspace/tables/get). The views and tables data sources follow the interface of the API exactly already (there, the API takes two parameters), so it makes sense to me to do the same for the table call.

Reply:
ah, Ok - I didn't check the API spec, sorry :-)